feat: adds an optional check to see if issuer property matches

kdenhartog opened the pull request:
This addresses issue #18

kdenhartog commented:
I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?

LoveishSarolia commented:
> I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?

If this answers your question, we use the ESLint extension in VSCode for base-level formatting.

kdenhartog-mattr committed:
feat: adds an optional check to see if issuer property matches

Signed-off-by: Kyle Den Hartog <kyle.denhartog@mattr.global>

kdenhartog commented:
> > I used VSCode's Javascript formatter, but it looks like it updated more than the code that I added. Do you guys normally use just 80 char text wrapping for formatting or some other tool?
> 
> If this answers your question, we use the ESLint extension in VSCode for base-level formatting.

Cleaned it up now following that format. Thanks for the tip.

mattcollier commented:
@kdenhartog thank you for the PR, we'll get some eyes on this soon, but it may take several days.

dmitrizagidulin reviewed (approved):
👍  from me, thanks @kdenhartog

dlongley commented on main.js:
```diff
@@ -181,6 +193,26 @@ async function _checkStatus({
     }
   }
 
+  /**
+   * This checks that the issuer of the verifiable credential matches
+   * the issuer of the revocationListCredential
+   */
```
We use this comment style for simple comments on conditionals, short code blocks, etc:

```suggestion
  // this checks that the issuer of the verifiable credential matches
  // the issuer of the revocationListCredential
```

The other style is reserved for jsdocs/descriptions of classes, functions, etc.

dlongley reviewed (approved):
LGTM -- can be merged after minor style fix on a comment.

dlongley commented:
Thanks, @kdenhartog!

davidlehn commented:
Our workflow setups are not running on external PRs.  Whoever plans to merge, please run test and lint scripts by hand first.

kdenhartog committed:
fix comment on 196-197 to match style guide

kdenhartog commented:
rebased the latest js style commit on and fixed the comment. Should be all ready to go now.

kdenhartog commented:
Any update needed here? @dmitrizagidulin @dlongley @mattcollier @gannan08

mattcollier commented:
Work on this continues here: https://github.com/digitalbazaar/vc-revocation-list/pull/23
